2022-02-07 14:01:00,676 INFO  ==============================================================
2022-02-07 14:01:00,677 INFO  Id:    log_request_num_lines
2022-02-07 14:01:00,678 INFO  Dir:   log_request_end_to_end/
2022-02-07 14:01:00,679 INFO  Title: Validate end to end behaviour for the log request operation check for number of lines
2022-02-07 14:01:00,679 DEBUG Execution order hint: 0.0
2022-02-07 14:01:00,680 INFO  ==============================================================
2022-02-07 14:01:00,685 DEBUG EnvironmentC8y Setup
2022-02-07 14:01:00,686 INFO  EnvironmentC8y Setup
2022-02-07 14:01:00,687 DEBUG Process parameters for systemctl<serv_mapper1>
  command line : /usr/bin/systemctl status tedge-mapper-c8y
    arg #1     : status
    arg #2     : tedge-mapper-c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/serv_mapper1.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/serv_mapper1.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:00,692 DEBUG Waiting up to 600 secs for process systemctl<serv_mapper1> (pid 13463)
2022-02-07 14:01:00,997 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-02-07 14:01:01,080 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
2022-02-07 14:01:01,081 INFO  Delaying execution by 7 seconds
2022-02-07 14:01:08,091 DEBUG Process parameters for sudo<tedge_connect_c8y>
  command line : /usr/bin/sudo /usr/bin/tedge connect c8y
    arg #1     : /usr/bin/tedge
    arg #2     : connect
    arg #3     : c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/tedge_connect_c8y.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/tedge_connect_c8y.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:08,100 DEBUG Waiting up to 600 secs for process sudo<tedge_connect_c8y> (pid 13470)
2022-02-07 14:01:19,471 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 11 secs
2022-02-07 14:01:19,474 DEBUG Process parameters for sudo<tedge_connect_c8y_test>
  command line : /usr/bin/sudo /usr/bin/tedge connect c8y --test
    arg #1     : /usr/bin/tedge
    arg #2     : connect
    arg #3     : c8y
    arg #4     : --test
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/tedge_connect_c8y_test.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/tedge_connect_c8y_test.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:19,481 DEBUG Waiting up to 600 secs for process sudo<tedge_connect_c8y_test> (pid 13616)
2022-02-07 14:01:19,838 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-02-07 14:01:19,839 DEBUG Process parameters for systemctl<serv_mosq2>
  command line : /usr/bin/systemctl status mosquitto
    arg #1     : status
    arg #2     : mosquitto
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/serv_mosq2.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/serv_mosq2.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:19,844 DEBUG Waiting up to 600 secs for process systemctl<serv_mosq2> (pid 13626)
2022-02-07 14:01:20,099 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-02-07 14:01:20,100 DEBUG Process parameters for systemctl<serv_mapper3>
  command line : /usr/bin/systemctl status tedge-mapper-c8y
    arg #1     : status
    arg #2     : tedge-mapper-c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/serv_mapper3.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/serv_mapper3.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:20,105 DEBUG Waiting up to 600 secs for process systemctl<serv_mapper3> (pid 13627)
2022-02-07 14:01:20,360 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-02-07 14:01:20,361 DEBUG Process parameters for sudo<rm_logs>
  command line : /usr/bin/sudo rm -rf /tmp/sw_logs
    arg #1     : rm
    arg #2     : -rf
    arg #3     : /tmp/sw_logs
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/rm_logs.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/rm_logs.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:20,366 DEBUG Waiting up to 600 secs for process sudo<rm_logs> (pid 13628)
2022-02-07 14:01:20,671 INFO  Executed sudo<rm_logs>, exit status 0
2022-02-07 14:01:20,688 DEBUG Process parameters for sudo<move_logs>
  command line : /usr/bin/sudo sh -c 'mv /tmp/sw_logs/* /var/log/tedge/agent/'
    arg #1     : sh
    arg #2     : -c
    arg #3     : mv /tmp/sw_logs/* /var/log/tedge/agent/
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/move_logs.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/move_logs.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:20,693 DEBUG Waiting up to 600 secs for process sudo<move_logs> (pid 13630)
2022-02-07 14:01:20,998 INFO  Executed sudo<move_logs>, exit status 0
2022-02-07 14:01:20,999 DEBUG --- test execute
2022-02-07 14:01:21,155 WARN  caught <class 'SystemError'> while running test: field id is missing in response
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1209, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/run.py", line 40, in execute
    log_file_request_payload, self.project.deviceid)
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/environments/cumulocity.py", line 115, in trigger_log_request
    raise SystemError("field id is missing in response")
SystemError: field id is missing in response
2022-02-07 14:01:21,167 WARN  SystemError: field id is missing in response ... blocked 
2022-02-07 14:01:21,167 DEBUG --- test cleanup
2022-02-07 14:01:21,168 INFO  
2022-02-07 14:01:21,168 INFO  cleanup:
2022-02-07 14:01:21,169 DEBUG Running registered cleanup function: <bound method LogRequestVerifyNumberOfLines.cleanup_logs of <LogRequestVerifyNumberOfLines object at 0x75cff730>>
2022-02-07 14:01:21,170 DEBUG Process parameters for sudo<rm_logs>
  command line : /usr/bin/sudo sh -c 'rm -rf /var/log/tedge/agent/example-*'
    arg #1     : sh
    arg #2     : -c
    arg #3     : rm -rf /var/log/tedge/agent/example-*
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/rm_logs.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/rm_logs.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:21,175 DEBUG Waiting up to 600 secs for process sudo<rm_logs> (pid 13641)
2022-02-07 14:01:21,480 INFO  Executed sudo<rm_logs>, exit status 0
2022-02-07 14:01:21,481 DEBUG Process parameters for sudo<rm_tmp_logs>
  command line : /usr/bin/sudo rm -rf /tmp/sw_logs
    arg #1     : rm
    arg #2     : -rf
    arg #3     : /tmp/sw_logs
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/rm_tmp_logs.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/rm_tmp_logs.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:21,486 DEBUG Waiting up to 600 secs for process sudo<rm_tmp_logs> (pid 13644)
2022-02-07 14:01:21,790 INFO  Executed sudo<rm_tmp_logs>, exit status 0
2022-02-07 14:01:21,792 DEBUG Process parameters for sudo<send_failed>
  command line : /usr/bin/sudo /usr/bin/tedge mqtt pub c8y/s/us 502,c8y_LogfileRequest
    arg #1     : /usr/bin/tedge
    arg #2     : mqtt
    arg #3     : pub
    arg #4     : c8y/s/us
    arg #5     : 502,c8y_LogfileRequest
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/send_failed.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/send_failed.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:21,797 DEBUG Waiting up to 600 secs for process sudo<send_failed> (pid 13646)
2022-02-07 14:01:22,085 INFO  Executed sudo<send_failed>, exit status 0
2022-02-07 14:01:22,086 DEBUG Running registered cleanup function: <bound method EnvironmentC8y.myenvcleanup of <LogRequestVerifyNumberOfLines object at 0x75cff730>>
2022-02-07 14:01:22,086 DEBUG EnvironmentC8y Cleanup
2022-02-07 14:01:22,163 DEBUG Process parameters for sudo<tedge_disconnect_c8y>
  command line : /usr/bin/sudo /usr/bin/tedge disconnect c8y
    arg #1     : /usr/bin/tedge
    arg #2     : disconnect
    arg #3     : c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/tedge_disconnect_c8y.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/tedge_disconnect_c8y.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:22,168 DEBUG Waiting up to 600 secs for process sudo<tedge_disconnect_c8y> (pid 13652)
2022-02-07 14:01:25,316 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-02-07 14:01:25,318 DEBUG Process parameters for systemctl<serv_mapper5>
  command line : /usr/bin/systemctl status tedge-mapper-c8y
    arg #1     : status
    arg #2     : tedge-mapper-c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/serv_mapper5.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/Output/linux/serv_mapper5.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-02-07 14:01:25,322 DEBUG Waiting up to 600 secs for process systemctl<serv_mapper5> (pid 13711)
2022-02-07 14:01:25,627 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-02-07 14:01:25,629 DEBUG ProcessUser cleanup function done.
2022-02-07 14:01:25,631 INFO  
2022-02-07 14:01:25,632 INFO  Test duration: 24.96 secs
2022-02-07 14:01:25,632 INFO  Test final outcome:  BLOCKED
2022-02-07 14:01:25,633 INFO  Test outcome reason: SystemError: field id is missing in response
2022-02-07 14:01:25,634 INFO  
